spec/components/support_interface/application_choice_component_spec.rb

```diff
@@ -72,15 +72,9 @@
           offer: build(:offer, conditions: conditions),
         )
       end
-
-      it 'does not render a link to the change the offered course choice when the`change_offered_course` flag is not active' do
```

Comment:
could we possibly repurpose this test to check for application choice to be in recruited state instead of removing?